app/queries/articles/active_threads_query.rb

```diff
@@ -0,0 +1,32 @@
+module Articles
+  class ActiveThreadsQuery
+    DEFAULT_OPTIONS = {
+      tags: ["discuss"],
+      time_ago: nil,
+      count: 10
+    }.with_indifferent_access.freeze
+
+    MINIMUM_SCORE = -4
+
+    def self.call(relation: Article.published, options: {})
+      options = DEFAULT_OPTIONS.merge(options)
+      tags, time_ago, count = options.values_at(:tags, :time_ago, :count)
+
+      relation.limit(count)
```

Comment:
**blocking**: this doesn't set the limit, I noticed because I currently have 4875 `discuss` articles in the sidebar 🤣 (I have lots of articles for the search PR)

```suggestion
      relation = relation.limit(count)
```

Reply:
Oh, whoops... 😅  Excellent catch!